packages/cli/src/services/project.service.ee.ts

```diff
@@ -42,7 +41,7 @@ export class TeamProjectOverQuotaError extends UserError {
 }
 
 export class UnlicensedProjectRoleError extends UserError {
-	constructor(role: ProjectRole | AssignableProjectRole) {
+	constructor(role: string) {
```

Comment:
I'm not sure why this is necessary? `AssignableProjectRole` should be all non empty strings that are not the personalProjectOwner role.